build.gradle.kts

```diff
@@ -111,6 +129,12 @@ tasks.withType<KotlinCompile>().configureEach {
     }
 }
 
+tasks.withType<KotlinNativeTest> {
+    // A few test with invalid floating point number fail due to issue in Kotlin native parser
+    // https://youtrack.jetbrains.com/issue/KT-69327
+    enabled = false
+}
+
```

Comment:
Could we conditionally disable just the affected tests rather than disabling all native tests?

Reply:
Sure, will do. I was waiting for #574 to be merged as it has some changes needed to run tests for native target